sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2Exec.scala

```diff
@@ -455,9 +452,12 @@ trait V2TableWriteExec extends V2CommandExec with UnaryExecNode with AdaptiveSpa
         }
       )
 
-      val operationMetrics = getOperationMetrics(query)
+      val writerMetrics = getWriterMetrics(query)
       logInfo(log"Data source write support ${MDC(LogKeys.BATCH_WRITE, batchWrite)} is committing.")
-      batchWrite.commit(messages, operationMetrics)
+      writerMetrics match {
```

Comment:
Will these metrics show up in the Spark SQL UI?
If yes, For the same V2 write operator, would show different metrics based on the operation?